dell-csi-helm-installer/csi-offline-bundle.md

```diff
@@ -46,84 +45,93 @@ To perform an offline installation of a driver or the Operator, the following st
 This needs to be performed on a linux system with access to the internet as a git repo will need to be cloned, and container images pulled from public registries.
 
 The build an offline bundle, the following steps are needed:
-1. Perform a `git clone` of the desired repository. For a helm based install, the specific driver repo should be cloned. For an Operator based deployment, the Dell CSI Operator repo should be cloned
-2. Run the `csi-offline-bundle.sh` script with an argument of `-c` in order to create an offline bundle
+1. Perform a `git clone` of the desired repository. For a helm based install, the specific driver repo should be cloned. For an Operator based deployment, the Dell CSM Operator repo should be cloned
```

Comment:
Change helm to HELM or Helm (use this same convention everywhere). Sorry, I know some of these are legacy and not really in the scope of this PR, but I think a search and replace might be easy to drive consistency.